systemtest/src/test/java/io/strimzi/systemtest/kafka/listeners/ListenersST.java

```diff
@@ -668,6 +668,102 @@ void testLoadBalancerTls(ExtensionContext extensionContext) {
         );
     }
 
+    @ParallelNamespaceTest
+    @Tag(INTERNAL_CLIENTS_USED)
+    void testClusterIp(ExtensionContext extensionContext) {
+        final TestStorage testStorage = new TestStorage(extensionContext);
+
+        final String namespaceName = testStorage.getNamespaceName();
+        final String clusterName = testStorage.getClusterName();
+        final String topicName = testStorage.getTopicName();
+        final String userName = testStorage.getUserName();
+        final String producerName = testStorage.getProducerName();
+        final String consumerName = testStorage.getConsumerName();
+
+        resourceManager.createResource(extensionContext, KafkaTemplates.kafkaEphemeral(clusterName, 3)
+            .editSpec()
+                .editKafka()
+                    .withListeners(new GenericKafkaListenerBuilder()
+                        .withName(Constants.CLUSTER_IP_LISTENER_DEFAULT_NAME)
+                        .withPort(9102)
+                        .withType(KafkaListenerType.CLUSTER_IP)
+                        .withTls(false)
+                        .build())
+                    .withConfig(Collections.singletonMap("default.replication.factor", 3))
+                .endKafka()
+            .endSpec()
+            .build());
+
+        ServiceUtils.waitUntilAddressIsReachable(KafkaResource.kafkaClient().inNamespace(namespaceName).withName(clusterName).get().getStatus().getListeners().get(0).getAddresses().get(0).getHost());
+
+        KafkaClients kafkaClients = new KafkaClientsBuilder()
+                .withNamespaceName(namespaceName)
+                .withTopicName(topicName)
+                .withBootstrapAddress(KafkaResources.bootstrapServiceName(testStorage.getClusterName()) + ":9122")
+                .withMessageCount(MESSAGE_COUNT)
+                .withUserName(userName)
+                .withProducerName(producerName)
+                .withConsumerName(consumerName)
+                .build();
+
+        resourceManager.createResource(extensionContext, kafkaClients.producerStrimzi());
+        ClientUtils.waitForClientSuccess(testStorage.getProducerName(), testStorage.getNamespaceName(), MESSAGE_COUNT);
+
+        resourceManager.createResource(extensionContext, kafkaClients.consumerStrimzi());
+        ClientUtils.waitForClientSuccess(testStorage.getConsumerName(), testStorage.getNamespaceName(), MESSAGE_COUNT);
+
+    }
+
+    @ParallelNamespaceTest
+    @Tag(SANITY)
```

Comment:
Can you please remove the `SANITY` tag here as well?

Reply:
aha :) I guessed it, but I figured if Lukas hadn't mentioned it in the last reviews, maybe we should keep it.

I'll remove it

Reply:
Sorry about that! My bad :)